src/test/java/seedu/address/testutil/TypicalProperties.java

```diff
@@ -39,6 +45,18 @@ public static PropertyBook getTypicalPropertyBook() {
         return propertyBook;
     }
 
+    /**
+     * Returns a {@code PropertyBook} with all the typical properties and client.
+     */
+    public static PropertyBook getTypicalPropertyBookWithClient() {
+        PropertyBook propertyBook = new PropertyBook();
+        for (Property property : getTypicalProperties()) {
+            propertyBook.addProperty(property);
+        }
+        propertyBook.addProperty(JURONG);
+        return propertyBook;
+    }
+
```

Comment:
Is there a reason for defining a separate method to get properties with clients? Will it be better to just include `JURONG` in the `getTypicalProperties` method?

Reply:
Doing that causes quite a number of tests to fail due to having hardcoded comparisons. Since most tests don't actually interact with Client or AskingPrice, it is safer to just implement a new method for properties with clients.